bin/pre-commit-hook.js

```diff
@@ -3,28 +3,78 @@
 const execSync = require( 'child_process' ).execSync;
 const spawnSync = require( 'child_process' ).spawnSync;
 const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const prettier = require( 'prettier' );
+const path = require( 'path' );
 
-console.log( '\nBy contributing to this project, you license the materials you contribute ' +
+console.log(
+	'\nBy contributing to this project, you license the materials you contribute ' +
 	'under the GNU General Public License v2 (or later). All materials must have ' +
-	'GPLv2 compatible licenses — see .github/CONTRIBUTING.md for details.\n\n' );
+	'GPLv2 compatible licenses — see .github/CONTRIBUTING.md for details.\n\n'
+);
 
 // Make quick pass over config files on every change
 require( '../server/config/validate-config-keys' );
 
 const files = execSync( 'git diff --cached --name-only' )
 	.toString()
 	.split( '\n' )
-	.map( ( name ) => name.trim() )
-	.filter( ( name ) => name.endsWith( '.js' ) || name.endsWith( '.jsx' ) );
+	.map( name => name.trim() )
+	.filter( name => name.endsWith( '.js' ) || name.endsWith( '.jsx' ) );
 
+/**
+ * Returns true if the given text contains '@ format' (without the space after @).
+ * within its first docblock. False otherwise.
+ *
+ * Note: I only don't write out the actual @ f o r m a t, because then this
+ * file would also get formatted!
```

Comment:
Good question! I plan on fixing this in a following pr, but right now our trailing commas are set to 'all' instead of 'es5'. [see here](https://github.com/Automattic/calypso-prettier/commit/af2cb116ff49fdcf0f8fc70a78119049c4a4a83f#diff-91ba5f4df12e1b6b0629662164b3f14eR14). Without prior transformations, node can't handle trailing commas in function calls.

invalid in node v6
```js
fn(
  arg,
);
```